src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+
+public class Event extends Task {
+    private String from;
+    private String to;
```

Comment:
Perhaps you want to make these 2 variable names more specific? the 'from' and 'to' might be ambiguous in the sense that what they are referring to, like time, or space..?